presto-native-execution/presto_cpp/main/operators/PartitionAndSerialize.cpp

```diff
@@ -45,6 +43,11 @@ void PartitionAndSerializeNode::addDetails(std::stringstream& stream) const {
 
 namespace {
 
+/// Partitions the input row based on partition function and serializes the
+/// entire row.
+/// The output of this operator is a row with 2 columns, with the
```

Comment:
> The output of this operator is a row with 2 columns, with the

The output has 2 columns: (1) partition number (INTEGER); (2) serialized row (VARBINARY).